wordpress-mobile/PasscodeLock-Android
# Update Layout Design

### Fix
Update the passcode layout for the new design.  Note that these changes only affect the layout of the library.  The colors and text can be overridden in the client app.  The sample app is shown in the ***Before*** and ***After*** screenshots while the ***Design*** mockups are for the Simplenote app, which will be updated in a subsequent pull request.  See the screenshots below for illustration.

![update_passcode_layout_phone](https://user-images.githubusercontent.com/3827611/69080103-8ec47300-09f8-11ea-90fb-7ed159e42bb1.png)

![update_passcode_layout_tablet](https://user-images.githubusercontent.com/3827611/69080107-908e3680-09f8-11ea-87d2-4bada7f1836c.png)

### Test
The fingerprint icon shown below the number 7 is only visible if the device has fingerprint authentication hardware.  If the device does not have that hardware, the icon is not shown.
1. Launch sample app.
2. Tap ***Settings*** action.
3. Tap ***Enable lock*** item.
4. Enter passcode.
5. Enter passcode again.
6. Notice app layout is as screenshots shown above.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
